app/assets/javascripts/html5_webshim.js

```diff
@@ -0,0 +1,15 @@
+$(document).on('ready page:load', function(){
+    window.webshim.setOptions('forms-ext', {
+    lazyCustomMessages: true,
+    replaceUI: true,
+    waitReady: true,
+    type: 'date',
+    date: {
+      startView: 2,
+      openOnFocus: true
+    }
+  });
+
+  window.webshim.polyfill('forms-ext');
+  window.onunload = function(){};
```

Comment:
Just out of curiosity, what does this empty `window.onunload` function do for us?